### DISCUSSION TITLE
Video FPS compared to particle velocity for PIV analysis

### DISCUSSION THREAD

**Discussion by e-riv** [OpenPIV/openpiv-python#314] [Q&A]
Hi,

I'm trying to figure out if the camera I have takes sufficiently high-speed video for analysing my experimental system. It seems this is going to be dependent on the speed at which the tracer particles are moving, but does anyone know what sort of ratio is typically need?

Thanks!

**Comment by eguvep** [comment]
Have a look at these two classical papers by Keane and Adrian:
DOI: 10.1088/0957-0233/1/11/013
DOI: 10.1088/0957-0233/2/10/013
They contain a thorough analysis and advices for choosing optimal PIV parameters. I hope, this helps.

**Reply by alexlib** [reply]
You can also send a couple of images to the openpiv-users forum on Google groups and get some experienced eyes on it.
